src/v1/Models/Common.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * FusionSuite - Backend
+ * Copyright (C) 2022 FusionSuite
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace App\v1\Models;
+
+class Common
+{
+  /**
+   * Get the user attributes for created_by, udpated_by and deleted_by.
+   */
+  public static function getUserAttributes($user_id)
+  {
+    if (is_null($user_id))
+    {
+      return null;
+    }
+    $user = \App\v1\Models\Item::find($user_id);
```

Comment:
what if `$user` is null?

Reply:
I can confirm it's an issue: after having deleted a user, I no longer can load my organizations because `find` returns null and it fails just after.

Reply:
I added code to return 'deleted user' name.